ui_src/src/domain/stationsList/stationBoxOverview/index.js

```diff
@@ -28,8 +28,12 @@ import replicasIcon from '../../../assets/images/replicasIcon.svg';
 import totalMsgIcon from '../../../assets/images/totalMsgIcon.svg';
 import poisonMsgIcon from '../../../assets/images/poisonMsgIcon.svg';
 import remoteStorage from '../../../assets/images/remoteStorage.svg';
+import { ReactComponent as ClockIcon } from '../../../assets/images/TimeFill.svg';
```

Comment:
You can import it in a more readable way:
import ClockIcon  from '../../../assets/images/TimeFill.svg';

Reply:
No, It does not work in that way. When used that way you cannot use it as standalone component. it is imported as path